scripts/conversionScripts/mpiLegacy.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2017 Battelle Energy Alliance, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import xml.etree.ElementTree as ET
+import xml.dom.minidom as pxml
+import os
+
+def convert(tree,fileName=None):
+  """
+    Converts input files to be compatible with merge request ###, where BasicStatistics is given the power
+    to be more nodalized than before.
+    @ In, tree, xml.etree.ElementTree.ElementTree object, the contents of a RAVEN input file
+    @ In, fileName, the name for the raven input file
+    @Out, tree, xml.etree.ElementTree.ElementTree object, the modified RAVEN input file
+  """
+  simulation = tree.getroot()
+  runinfo = simulation.find('RunInfo')
+  if runinfo is None:
+    return tree #No RunInfo, no need to change.
+  if runinfo.find('mode') is not None:
+    mode = runinfo.find('mode')
+    foundLegacy = False
+    for child in mode:
+      if child.tag.lower() == "nosplitnode":
+        foundLegacy = True
+      if child.tag.lower() == "limitnode":
+        foundLegacy = True
+    print(foundLegacy)
```

Comment:
Are these print statements for debugging? Or should they remain in this conversion script to ensure that the script is doing what the user expects?